src/checkout/checkout-service.ts

```diff
@@ -1052,6 +1054,25 @@ export default class CheckoutService {
         return this._dispatch(action);
     }
 
+    /**
+     * Initializes the spam protection for order creation.
+     *
+     * The order creation will fail if spam protection is enabled but
+     * is not completed.
+     *
+     * ```js
+     * await service.initializeOrderSpamProtection('spamProtectionContainer');
+     * ```
+     *
+     * @param containerId - The container ID where the spam protection should be initialized.
```

Comment:
`The container ID where the spam protection should be rendered.`  ?

Reply:
👍